docs/reference/logging/overview.md

```diff
@@ -41,6 +43,21 @@ Logs that reach `max-size` are rolled over. You can set the size in kilobytes(k)
 
 If `max-size` and `max-file` are set, `docker logs` only returns the log lines from the newest log file.
 
+The `labels` and `env` options adds additional attributes to be logged
```

Comment:
Yeah definitely better :+1:. I'll update (except for the `docker inspect` part, where I'll probably point out where the log is)